core/templates/dev/head/pages/topic_editor/questions/QuestionsTabDirective.js

```diff
@@ -42,6 +45,10 @@ oppia.directive('questionsTab', [
             $scope.canEditQuestion = $scope.topicRights.canEditTopic();
             $scope.questionSummaries =
               TopicEditorStateService.getQuestionSummaries();
+            $scope.emptyMisconceptionsList = [];
```

Comment:
I get that this starts off empty. But will it be empty forever? Wouldn't people who suggest questions be adding misconceptions to them too?

Reply:
@nithusha21 also, there doesn't seem to be a provision to add misconception in the UI.

Reply:
For the initial version of this feature, I have allowed the question suggestion creator to select a topic, and then the reviewer must link the question to a skill. To tag answers with misconceptions, we need to know which skill is (going to be) linked with the question.  However, at the point where the reviewer links the question to a skill, there is no provision to update the question (yet). So I think once we create a way to update suggestions once they have been submitted, I can add this feature. But I have excluded it in the first version of the questions suggestion feature.

Reply:
So, the onus is on the reviewer to link a misconception? 

@seanlip On a side note, I'm still not very clear with the concept of "misconception" and how it is linked to question or a skill. As a creator, I feel pretty confused about these new terminologies (i.e. new structures) and I'm slightly concerned that creating lessons has a steep learning curve now (or maybe I just don't understand it). Having tried creating questions, topics and skills extensively, I have a few things to discuss with you post GSoC.

Reply:
Yes, I'll file an issue for this. From the current UX, the reviewer links a skill (and only then misconceptions are "defined", as misconceptions link to skills). So I guess it would be the responsibility of the reviewer finally.